alphagov/paas-cf
# [#155623416] Restructure/rename instance_groups to match cf-deployment (PR 1 of 4)

[#155623416 Restructure/rename jobs to match cf-deployment](https://www.pivotaltracker.com/story/show/155623416)

What?
-----

We want to refactor our manifest to be based on the upstream [cf-deployment](https://github.com/cloudfoundry/cf-deployment). In this PR we move closer to this direction by:

 - Adopting the bosh v2 instance_group/jobs convention rather than jobs/templates.
 - Renaming the instance groups as they are called in cf-deployment
 - Reorganising the jobs within the different instance_groups to match the cf-deployment setup.

We had to split this change in two different PRs, as moving services between instance groups in one go caused downtime. In other PR we remove the old unnecessary instance_groups and we do other changes.

**NOTE**: We decided delay the migration of the route_emitter to a follow up PR.

How to review?
--------------

 - Code review
 - Deploy the master branch
 - Deploy this branch

There should not be any downtime in the application availability test. We spec one or two errors on the API deployment, but not a lot.

Who?
---

Anyone but @keymon or @henrytk
